Modules/booking/Controller/BookinginvoiceController.php

```diff
@@ -363,20 +348,29 @@ protected function invoice($id_space, $beginPeriod, $endPeriod, $id_resp, $numbe
             }
 
             $userTime = array();
-            //print_r($reservations);
             $userTime["nb_hours_day"] = 0;
             $userTime["nb_hours_night"] = 0;
             $userTime["nb_hours_we"] = 0;
+            $userTime["res_quantity"] = 0;
             foreach ($reservations as $reservation) {
 
                 // count: day night we, packages
                 if ($reservation["package_id"] > 0) {
                     $userPackages[$reservation["package_id"]] ++;
                 } else {
                     $resaDayNightWe = $this->calculateTimeResDayNightWe($reservation, $timePrices[$res["id"]]);
-                    $userTime["nb_hours_day"] += $resaDayNightWe["nb_hours_day"];
-                    $userTime["nb_hours_night"] += $resaDayNightWe["nb_hours_night"];
-                    $userTime["nb_hours_we"] += $resaDayNightWe["nb_hours_we"];
+                    
+                    /* without taking resource quantity into account
+                    * $userTime["nb_hours_day"] += $resaDayNightWe["nb_hours_day"];
+                    * $userTime["nb_hours_night"] += $resaDayNightWe["nb_hours_night"];
+                    * $userTime["nb_hours_we"] += $resaDayNightWe["nb_hours_we"];
+                    */
+
+                    // taking resource quantity into account
+                    $qte = $resaDayNightWe["res_quantity"];
```

Comment:
not sure quantities should be multiplied by duration, should be checked to understand how it is expected to work